pages/establishment/pdf/views/establishment.jsx

```diff
@@ -0,0 +1,196 @@
+import React, { Fragment } from 'react';
+import ReactMarkdown from 'react-markdown';
+import map from 'lodash/map';
+import StandardConditions from '../../../common/views/pdf/standard-conditions';
+
+const getName = person => `${person.firstName} ${person.lastName}`;
+
+const triggerContinued = () => {
+  let first = true;
+
+  return () => {
+    if (first) {
+      first = false;
+      return '';
+    }
+    return ' (continued)';
+  };
+};
+
+const Establishment = ({ establishment, content }) => {
+  const licenceHolders = establishment.namedPeople.pelh || establishment.namedPeople.nprc || [];
```

Comment:
Based on what you told me about the reasons for not using `groupBy` in the `namedPeople` function then `establishment.namedPeople.pelh` will always be truthy since it will be `[]` even if no PEL holder roles exist.

Reply:
```suggestion
  const licenceHolders = [].concat(establishment.namedPeople.pelh).concat(establishment.namedPeople.nprc);
```

Reply:
That said - I'm not sure it's legit to describe an NPRC as a licence holder.

Reply:
My instinct would be to only ever call explicit PEL holders "licence holders", and just display "None" if there are none.

We have the section for NPRCs under named people already so we're covered there.

Reply:
So in fact...
```suggestion
  const licenceHolders = establishment.namedPeople.pelh;
```